### PR TITLE
translaton(yicong): add dmc2gym english file

### PR DESCRIPTION
**Pull request by sylviai**

### CODE REVIEW

File: source/13_envs/dmc2gym.rst [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
++------------+----------+------------+------------+-----------+
+|reacher     |easy      |4           |2           |7          |
++------------+----------+------------+------------+-----------+
+|walker      |walk      |18          |6           |24         |
++------------+----------+------------+------------+-----------+
+
+.. note::
+   The task in dm_control follows the Markov Decision Process (MDP).
+
+    - State \ ``s``\ is a real vector except for the spatial direction: math:`\cal{S} \equiv \mathbb{R}^{dim(\cal{S})}` ，where the spatial direction is represented by Unit quaternion: math:`\in SU(2)`
```
Comment (karroyan) [inline source/13_envs/dmc2gym.rst:153]: The math formula isn't shown properly

File: source/13_envs/dmc2gym.rst [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
++------------+----------+------------+------------+-----------+
+|walker      |walk      |18          |6           |24         |
++------------+----------+------------+------------+-----------+
+
+.. note::
+   The task in dm_control follows the Markov Decision Process (MDP).
+
+    - State \ ``s``\ is a real vector except for the spatial direction: math:`\cal{S} \equiv \mathbb{R}^{dim(\cal{S})}` ，where the spatial direction is represented by Unit quaternion: math:`\in SU(2)` 
+
+    - Observation \ ``o(s, a)``\ describes the observation result to the agent. The tasks we implement are all strongly observable, that is, the state can be recovered from a single observation. The observed features that depend only on the state (position and velocity) are a function of the current state. Observations that also depend on controls (such as touch sensor readings) are functions of previous transitions.
```
Comment (karroyan) [inline source/13_envs/dmc2gym.rst:155]: Observation describes the state that the agent can observe.

File: source/13_envs/dmc2gym.rst [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+Based on Image Observation
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  When setting \ ``from_pixels=True``\ ，the observation space is a three-channel game image with height and width respectively.
+
+-  The size of the observed image can be adjusted by setting the \ ``height, width``\ parameters.
+
+-  Observe the specific shape of the observation space by setting \ ``channels_first``\ .
+
+   -  \ ``channels_first=True``\ , observation shape is \ ``[3, height, width]``\
```
Comment (karroyan) [inline source/13_envs/dmc2gym.rst:169]: ，-> ,

File: source/13_envs/dmc2gym.rst [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+
+   -  \ ``channels_first=False``\ ，observation shape is \ ``[height, width, 3]``\
+
+-  The range of a single pixel value for each channel is \ ``[0, 255]``\ ， and the data type is \ ``uint8``\
+
+Non-Image Based Observation 
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  when \ ``from_pixels=False``\ is set，the viewing space dimension follows \ ``dim(O)``\ in the above table
```
Comment (karroyan) [inline source/13_envs/dmc2gym.rst:178]: when->When
the viewing space -> the observation space

File: source/13_envs/dmc2gym.rst [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+
+
+Random Seeds
+--------
+
+-  There are two parts of random seeds in the environment that need to be set, one is the random seed of the original environment, and the other is the random seed of the random library used by various environment transformations（such as \ ``random``\ ，\ ``np.random``\)
+
+-  For the environment caller, just set two seeds through the\ ``seed``\ method of the environment，and do not need to care about the specific implementation details.
+
+-  Concrete implementation inside the environment: For the seed of the original environment, set before calling the \ ``reset``\ method of the environment ，before the concrete \ ``reset``\
```
Comment (karroyan) [inline source/13_envs/dmc2gym.rst:227]: set before calling the \ ``reset``\ method of the environment ，before the concrete \ ``reset``\ -> set in the reset methods of the environment calling function, before the concrete environment implementation reset

File: source/13_envs/dmc2gym.rst [hunk 6]
```diff
@@ -0,0 +1,336 @@
...
+--------
+
+-  There are two parts of random seeds in the environment that need to be set, one is the random seed of the original environment, and the other is the random seed of the random library used by various environment transformations（such as \ ``random``\ ，\ ``np.random``\)
+
+-  For the environment caller, just set two seeds through the\ ``seed``\ method of the environment，and do not need to care about the specific implementation details.
+
+-  Concrete implementation inside the environment: For the seed of the original environment, set before calling the \ ``reset``\ method of the environment ，before the concrete \ ``reset``\ 
+
+-  The specific implementation inside the environment: for random library seeds, set the value directly in the \ ``seed``\ method of the environment ; for the seed of the original environment, inside the \ ``reset``\ method of the calling environment, the specific original environment \ ``reset``\ was previously set to seed + np_seed, where seed is the value of the aforementioned random library seed,
+np_seed = 100 * np.random.randint(1, 1000).
```
Comment (karroyan) [inline source/13_envs/dmc2gym.rst:230]: duplicate

**Review by karroyan — CHANGES_REQUESTED** [review]

File: source/13_envs/dmc2gym.rst (hunk 6 above)
Comment (sylviai) [reply source/13_envs/dmc2gym.rst:233]: 红色部分是删去的，绿色部分是修改过的，没有重复

**Pull request merged**